packages/state-transition/src/util/slot.ts

```diff
@@ -21,7 +21,27 @@ export function computeTimeAtSlot(config: ChainConfig, slot: Slot, genesisTime:
   return genesisTime + slot * config.SECONDS_PER_SLOT;
 }
 
-export function getCurrentInterval(config: ChainConfig, secondsIntoSlot: number): number {
-  const timePerInterval = Math.floor(config.SECONDS_PER_SLOT / INTERVALS_PER_SLOT);
-  return Math.floor(secondsIntoSlot / timePerInterval);
+export function getAttestationDueMs(config: ChainConfig): number {
```

Comment:
can we already pass epoch here, this reduces the diff later on for ePBS

```suggestion
export function getAttestationDueMs(config: ChainConfig, _epoch: Epoch): number {
```

Reply:
I think we can do `_fork: ForkName` instead

Reply:
good point, makes sense to do that instead